src/controllers/authController.js

```diff
@@ -55,8 +112,25 @@ const login = async (req, res) => {
 };
 
 //logout endpoint
-
 const logout = async (req, res) => {
+  /*
+    #swagger.summary = 'Logout a user'
+    #swagger.parameters['token'] = {
+        in: 'cookie',
+        description: 'JWT token in the cookie',
+        required: true,
+        type: 'string'
+    }
+    #swagger.responses[200] = {
+        description: 'User logged out successfully',
+        schema: { type: 'object', properties: { msg: { type: 'string' } } }
+    }
+    #swagger.security = [{ "JWT": [] }]  // Specify that JWT token is required for access
+    #swagger.responses[401] = {
+        description: 'Unauthorized',
+        schema: { type: 'object', properties: { msg: { type: 'string' } } }
+    }
```

Comment:
I'm not seeing in the logout function code in which case we return a 401?

Reply:
Removed